cmake/setup_write_config.cmake

```diff
@@ -224,13 +224,6 @@ FOREACH(_feature ${_deal_ii_features_sorted})
     IF(_feature MATCHES "THREADS" AND DEFINED TBB_VERSION)
       _detailed("#            TBB_VERSION = ${TBB_VERSION}\n")
     ENDIF()
-    IF(_feature MATCHES "CUDA" AND DEFINED CUDA_COMPUTE_CAPABILITY)
-      _detailed("#            CMAKE_CUDA_COMPILER = ${CMAKE_CUDA_COMPILER}\n")
-      _detailed("#            CUDA_COMPUTE_CAPABILITY = ${CUDA_COMPUTE_CAPABILITY_MAJOR}.${CUDA_COMPUTE_CAPABILITY_MINOR}\n")
```

Comment:
This would still be nice to know. Do you know if Kokkos has some kind of information available that we can show?

Reply:
We could print the enabled `Kokkos` backends and architectures.